x-pack/plugins/security_solution/cypress/integration/detection_alerts/changing_alert_status.spec.ts

```diff
@@ -0,0 +1,366 @@
+          selectNumberOfAlerts(numberOfAlertsToBeSelected);
+          cy.get(TAKE_ACTION_POPOVER_BTN).should('exist');
+
+          closeFirstAlert();
+          waitForAlerts();
+
+          const expectedNumberOfAlerts = +numberOfAlerts - numberOfAlertsToBeClosed;
+          cy.get(ALERTS_COUNT).should('have.text', `${expectedNumberOfAlerts} alerts`);
+          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should(
+            'have.text',
+            `${expectedNumberOfAlerts}`
+          );
+
+          goToClosedAlerts();
+          waitForAlerts();
+
+          cy.get(ALERTS_COUNT).should('have.text', `${numberOfAlertsToBeClosed} alert`);
+          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should(
+            'have.text',
+            `${numberOfAlertsToBeClosed}`
+          );
+        });
+    });
+
+    it('Updates trend histogram whenever alert status is updated in table', () => {
+      const numberOfAlertsToBeClosed = 1;
+      cy.get(ALERTS_COUNT)
+        .invoke('text')
+        .then((alertNumberString) => {
+          const numberOfAlerts = alertNumberString.split(' ')[0];
+          cy.get(ALERTS_COUNT).should('have.text', `${numberOfAlerts} alerts`);
+          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should('have.text', `${numberOfAlerts}`);
+
+          selectNumberOfAlerts(numberOfAlertsToBeClosed);
+
+          cy.get(SELECTED_ALERTS).should('have.text', `Selected ${numberOfAlertsToBeClosed} alert`);
+
+          closeAlerts();
+          waitForAlerts();
+
+          const expectedNumberOfAlertsAfterClosing = +numberOfAlerts - numberOfAlertsToBeClosed;
+          cy.get(ALERTS_COUNT).should('have.text', `${expectedNumberOfAlertsAfterClosing} alerts`);
+          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should(
+            'have.text',
+            `${expectedNumberOfAlertsAfterClosing}`
+          );
+
+          goToClosedAlerts();
+          waitForAlerts();
+
+          cy.get(ALERTS_COUNT).should('have.text', `${numberOfAlertsToBeClosed} alert`);
+
+          const numberOfAlertsToBeOpened = 1;
+          selectNumberOfAlerts(numberOfAlertsToBeOpened);
+
+          cy.get(SELECTED_ALERTS).should('have.text', `Selected ${numberOfAlertsToBeOpened} alert`);
+          cy.get(ALERTS_TREND_SIGNAL_RULE_NAME_PANEL).should('exist');
+
+          openAlerts();
+          waitForAlerts();
+
+          cy.get(ALERTS_COUNT).should('not.exist');
+          cy.get(ALERT_COUNT_TABLE_FIRST_ROW_COUNT).should('not.exist');
+          cy.get(ALERTS_TREND_SIGNAL_RULE_NAME_PANEL).should('not.exist');
+        });
+    });
+  });
+
+  context('Changing alert status with read only role', () => {
+    before(() => {
+      login(ROLES.t2_analyst);
+    });
+    beforeEach(() => {
+      deleteAlertsAndRules();
+      createCustomRuleEnabled(getNewRule());
+      visit(ALERTS_URL);
+      waitForAlertsToPopulate();
+    });
+    it('Mark one alert as acknowledged when more than one open alerts are selected', () => {
```

Comment:
nit: new line above and below the `before`, `beforeEach`, and `it`. Feel free to ignore as this is a large PR.